StripePaymentSheet/StripePaymentSheet/Resources/Localizations/en.lproj/Localizable.strings

```diff
@@ -34,6 +34,9 @@
 /* Text for back button */
 "Back" = "Back";
 
+/* A label used in various UIs, including a button that represents a payment method type 'Bank' - where a user can pay with their bank account instead of, say, a credit card. */
+"Bank" = "Bank";
+
```

Comment:
@yuki-stripe this string moved from `StripePayments`; I believe I might need to do something extra to localize?

https://github.com/stripe/stripe-ios/pull/3800/files#diff-8092afe7db972886c688e6ad1eb1426cf187d89047ad9074d2f669598a0ee15eL19-L21

Reply:
If the check-translations job passes (which it did) I think you're good to go.